include/frg/logging.hpp

```diff
@@ -135,6 +135,42 @@ constexpr auto output_to(Container &cont) {
 	return container_logger{cont};
 }
 
+#ifdef FRIGG_HAS_IOSTREAM
+	template<typename Out>
+	struct ostream_out {
```

Comment:
Preprocessor macros shouldn't have an effect on indentation of code and vice versa, please dedent